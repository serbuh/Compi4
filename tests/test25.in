// Foreach with computation and multiple use of the same counter

int count = (3 + 5*3)/9 + 11; //
C c_a = 100 C;
C c_b = 200 C;
F f_a = 100 F;
F f_b = 200 F;
K k_a = 100 K;
K k_b = 200 K;

foreach(count C) + 20;
print c_a;
print c_b;
print f_a;
print f_b;
print k_a;
print k_b;

foreach(count K) - 5;
print c_a;
print c_b;
print f_a;
print f_b;
print k_a;
print k_b;
