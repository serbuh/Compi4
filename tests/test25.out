360
460
100
200
100
200
360
460
100
200
35
135
